Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -368,9 +368,11 @@ void CClientVehicle::SetPosition(const CVector& vecPosition, bool bResetInterpol
     }
 
     // If we have any occupants, update their positions
+    // Make sure we dont update their position if they are getting out and have physically left the car
     for (int i = 0; i <= NUMELMS(m_pPassengers); i++)
         if (CClientPed* pOccupant = GetOccupant(i))
-            pOccupant->SetPosition(vecPosition);
+            if (pOccupant->GetVehicleInOutState() != VEHICLE_INOUT_GETTING_OUT || pOccupant->GetRealOccupiedVehicle())
+                pOccupant->SetPosition(vecPosition);
```

Comment:
When vehicle position is updated, we dont want to update position for peds who are exiting, because we removed the check in `CClientPed::SetPosition` (see above comment).